### DISCUSSION TITLE
USB-MIDI on High-Speed USB?

### DISCUSSION THREAD

**Discussion by aspdigital** [hathach/tinyusb#1010] [Q&A]
I've implemented USB MIDI on a handful of architectures -- EFM8UB2, EFM32GG, TM4C1294 -- using the vendors' libraries. (It's become a sort of "Hello, World!" first project on a new architecture.) I'm interested in migrating to tinyusb for the obvious reason: write once, use everywhere, especially since the LPC55 and LPC546 devices look great but NXP's USB stack is impenetrable and not really documented. 

I'm working on a design that requires High Speed USB (composite audio streaming and MIDI), so to that end, does tinyusb's USB MIDI implementation work at High Speed, at least on the LPC546 or LPC55?

**Comment by hathach** [comment]
yes, it should.
